lib/pennyworth.rb

```diff
@@ -28,35 +28,38 @@
 require "find"
 require "open-uri"
 
-require_relative "version"
-require_relative "exceptions"
-require_relative "cli"
-require_relative "helper"
-require_relative "vagrant"
-require_relative "commands/command"
-require_relative "commands/setup_command"
-require_relative "commands/status_command"
-require_relative "commands/base_command"
-require_relative "commands/build_base_command"
-require_relative "commands/import_base_command"
-require_relative "commands/up_command"
-require_relative "commands/down_command"
-require_relative "commands/boot_command"
-require_relative "commands/shutdown_command"
-require_relative "commands/import_ssh_keys_command"
-require_relative "commands/list_command"
-require_relative "vagrant_command"
-require_relative "settings"
-require_relative "runner"
-require_relative "image_runner"
-require_relative "vm"
-require_relative "pennyworth_libvirt"
-require_relative "ssh_keys_importer"
-require_relative "urls"
-require_relative "host_runner"
-require_relative "cli_host_controller"
-require_relative "host_config"
-require_relative "lock_service"
-require_relative "remote_command_runner"
-require_relative "local_runner"
-require_relative "local_command_runner"
+module Pennyworth
+end
+
+require "pennyworth/version"
```

Comment:
This change breaks loading the pennyworth library locally without being pulled in as a gem, so I think using `require_relative` makes sense here.